html-css/week1/index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <title>S J H </title>
+</head>
+<body>
+    <header>
+        <nav>
+            <img id="profile-pic" src="./images/me.jpeg" alt="profile-sjh"/>
+            <div class="nav-container">
+                
+                <div id="nav-home" class="nav-item">
```

Comment:
Excellent re-use of CSS through `nav-home`! 🚀 However, we should strive for IDs to unique identify elements thus having multiple elements with the same ID is not good. There is also issues with this for `square`, `pro-item`, and `test-item`.

Can you think of an alternative method to reuse styling that avoids elements sharing IDs? 😄

Reply:
I should have used class. 
thanks for pointing it out.